client-app/modules/quotes/pages/quotes.vue

```diff
@@ -55,11 +55,8 @@
     <VcEmptyView
       v-if="!fetching && !quotes.length"
       :text="$t(!!keyword ? 'pages.account.quotes.no_results_message' : 'pages.account.quotes.no_quotes_message')"
-    >
-      <template #icon>
-        <VcImage src="/static/images/common/order.svg" :alt="$t('pages.account.orders.no_orders_img_alt')" />
-      </template>
-    </VcEmptyView>
+      icon="thin-quotes"
```

Comment:
Icon name should not be related to services. Make the neutral name for the icon or move custom icon to module.